dbus-service.py

```diff
@@ -30,8 +30,8 @@
 
 from pyndn import Name, Face
 
-from eosdatadistribution.ndn import Consumer
```

Comment:
You changed the imports back to eos_data_distribution but the files are still named eosdatadistribution in this commit.

See the tree: https://github.com/endlessm/endless-ndn/tree/49266c8b01291b7feb74d051160d848554f454b7